File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Programming/LoadHandlingPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.Form.Programming
{
    public partial class LoadHandlingPage : UserControl
    {
        public LoadHandlingPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Click event of the button.
        /// Opens the demonstrated Form.
        /// </summary>
        private void mobjButton_Click(object sender, EventArgs e)
        {
            
            LoadEventForm form = new LoadEventForm();

            form.ShowDialog();
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Programming/LoadHandlingPage.vb
Namespace CompanionKit.Controls.Form.Programming

	Public Class LoadHandlingPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles Click event of the button.
        ''' Opens the demonstrated Form.
        ''' </summary>
        Private Sub mobjButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjButton.Click
            Dim form As New LoadEventForm()
            form.ShowDialog()
        End Sub
	End Class

End Namespace